packages/by-name/cloud-api-adaptor/package.nix

```diff
@@ -31,23 +34,32 @@ in
 
 buildGoModule rec {
   pname = "cloud-api-adaptor";
-  version = "0.9.0";
+  version = "0.10.0";
+
+  src = applyPatches {
+    src = fetchFromGitHub {
+      owner = "confidential-containers";
+      repo = "cloud-api-adaptor";
+      rev = "v${version}";
+      hash = "sha256-OaSIO26nlkeI2olSx0o8xdwhLMZ8eH753pUbyHypI+E=";
+    };
 
-  src = fetchFromGitHub {
-    owner = "confidential-containers";
-    repo = "cloud-api-adaptor";
-    rev = "v${version}";
-    hash = "sha256-5tDG0sEiRAsb259lPui5ntR6DVHDdcXhb04UESJzHhE=";
+    patches = [
+      # This fixes a route setting problem we see with our NixOS image that
+      # does not seem to occur with the upstream image.
+      # TODO(burgerdev): upstream
+      ./0001-netops-replace-routes-instead-of-adding-them.patch
+    ];
   };
 
   sourceRoot = "${src.name}/src/cloud-api-adaptor";
 
   proxyVendor = true;
-  vendorHash = "sha256-kqzi7jRF3tQ4/yLkJXfZly4EvVKFb400/WXlN0WjYm8=";
+  vendorHash = "sha256-FsckYZAiBfTEp25+dDNqPpB/550NqeEsutWC34s+GmE=";
 
-  nativeBuildInputs = lib.optional withLibvirt pkg-config;
+  nativeBuildInputs = [ makeWrapper ] ++ lib.optional withLibvirt pkg-config;
 
-  buildInputs = lib.optional withLibvirt libvirt;
+  buildInputs = [ pkgs.iptables ] ++ lib.optional withLibvirt libvirt;
```

Comment:
If you reference `iptables` in the `postInstall` below, that's enough to add it to the closure of the package, I think. You shouldn't need it here too.

Reply:
Done